aodntools/timeseries_products/Documentation/Hourly_timeseries.md

```diff
@@ -0,0 +1,157 @@
+- Variables to be aggregated do not represent current velocity.
+- The in-water data are bounded by the global attributes time_deployment_start and time_deployment_end.
+- Variables have been quality-controlled and flagged using the IMOS standard flags (see IMOS NetCDF Conventions v1.4, p43, Reference Table B)
+
+This project will not make any corrections to or implement work-arounds for input files that do not meet these requirements. A report on files that cannot be aggregated will be produced. If any files are re-processed by the moorings facility to meet the requirements, these will be incorporated into future versions of the product.
+
+## Method
+
+### Input file selection
+
+The input files can be accessed on the local file system, or remotely via the AODN THREDDS server using the OPeNDAP protocol.  A list of OPeNDAP URLs can be produced by the utility function get_moorings_url(), which queries an index of all published moorings files to obtain the files relevant for a given site. The list can also be filtered by feature type (e.g time-series or profile) temporal range, file version (FV01), processing mode (real-time/delayed) or a regular expression matched against the file name.
+
+
+
+### Input file validation
+
+Before proceeding to the aggregation, each input file will be checked to ensure it meets the requirements (as specified above under Inputs). This will prevent errors during the aggregation process. Any input files that fail to meet the requirements will be excluded from the aggregation, and their URL listed in a global attribute rejected_files. 
+
+### Dimensions
+
+The dimensions of the resulting file  are determined as follows:
+
+- `OBSERVATION`:  the total number of observation records, excluding out-of-the-water data, in all input files;
+- `INSTRUMENT`: the number of instruments (i.e. number of files); 
+- `strlen`: a fixed dimension of length 256 for string variables.
+
+
+### Variables
+
+The product will aggregate the following  non-velocity variables (standard/long name in brackets):
+
+`TEMP` (sea_water_temperature);  
+`PSAL` (sea_water_salinity);  
+`CPHL`, `CHLF`, `CHLU` (mass_concentration_of_inferred_chlorophyll_from_relative_fluorescence_units_in_sea_water) - considered as separate variables for now;  
+ `TURB` (sea_water_turbidity);  
+ `DOX1` (mole_concentration_of_dissolved_molecular_oxygen_in_sea_water);   
+`DOX2` (moles_of_oxygen_per_unit_mass_in_sea_water);  
+ `DOXS` (fractional_saturation_of_oxygen_in_sea_water);  
+ `PAR` (downwelling_photosynthetic_photon_flux_in_sea_water).  
+  
+
+
+
+Each of the VoIs are produced by selecting only the “good” and “probably good” data (QC flags 1 and 2), binning the values to a one-hour interval, and concatenating the resulting values into a single file. The resulting variables have dimension OBSERVATION. The VoI’s ancillary_variables, in particular the corresponding quality-control flags, are not included. 
```

Comment:
This is the first use of the abbreviation "VoI", so we need to spell it out at least here.